simulator/simulator.go

```diff
@@ -56,6 +57,10 @@ func startSimulator() error {
 			return xerrors.Errorf("creates a new Clientset for the ExternalKubeClientCfg: %w", err)
 		}
 	}
+	// Copy config before overriding it.
+	extendersCfgForSimulator := extender.CopyExtenderCfg(cfg.InitialSchedulerCfg.Extenders)
+	// Override the config so that the communication of the Extender goes to the simulator.
+	extender.OverrideExtendersCfgToSimulator(cfg.InitialSchedulerCfg, cfg.Port)
```

Comment:
`OverrideExtendersCfgToSimulator` is only called here. What if the scheduler configuration is changed via the API (= from web ui)?

Reply:
I guess we need to execute `OverrideExtendersCfgToSimulator` around https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/6dd1a4d3086b1a1fb13c1f020a4a2b2705646474/simulator/scheduler/scheduler.go#L160